frontend/src/hooks/api/actions/useAdminUsersApi/useAdminUsersApi.ts

```diff
@@ -16,10 +10,7 @@ export const REMOVE_USER_ERROR = 'removeUser';
 
 const useAdminUsersApi = () => {
     const { loading, makeRequest, createRequest, errors } = useAPI({
-        handleBadRequest,
-        handleNotFound,
-        handleUnauthorized,
-        handleForbidden,
+        propagateErrors: true,
```

Comment:
The reason for this change was to have proper errors in the UI toast, otherwise we would get `cannot read property "msg" of undefined` errors. I think this is a safe change. Can you please confirm @FredrikOseberg and @Tymek?

Reply:
Double checked with @FredrikOseberg and this should be safe as long as the calls to this hook's methods are wrapped in a `try`/`catch`, which they seem to be.